### PR TITLE
Implement all the features of stores command in store

### PR DESCRIPTION
**Pull request by shafreeck**
### What problem does this PR solve? 

Unify the `store` and `stores` commands

### What is changed and how it works?

* Mark `stores` command as deprecated
* `store remove-tombstone`  to remove stores in tombstone state. Replace `stores remove-tombstone`
* `store limit` show all the store limits. Replace `stores show limit`
* `store limit all <rate>` set rate for all stores. Replace `stores limit <rate>`
* Enable the `--help/-h` flag for subcommand.(Ex. pdctl store -h prints help message for store other than pdctl)
* Unify all the help message with cobra's `Usage()` function.

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to update the documentation
 - Need to be included in the release notes

### CODE REVIEW

File: tools/pd-ctl/pdctl/command/store_command.go [hunk 1]
```diff
@@ -31,14 +31,15 @@ var (
 // NewStoreCommand return a stores subcommand of rootCmd
 func NewStoreCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   `store [delete|label|weight|limit] <store_id> [--jq="<query string>"]`,
-		Short: "show the store status",
+		Use:   `store [comnand] [flags]`,
```
Comment (lhy1024) [inline tools/pd-ctl/pdctl/command/store_command.go]:
```suggestion
		Use:   `store [command] [flags]`,
```

Comment (lhy1024) [inline tools/pd-ctl/pdctl/command/store_command.go]:
```suggestion
		Use:   `store [command] [flags]`,
```

File: tools/pd-ctl/main.go [hunk 2]
```diff
@@ -59,10 +59,6 @@ func main() {
 	flag.CommandLine.ParseErrorsWhitelist.UnknownFlags = true
 	flag.Parse()
 
-	if help {
```
Comment (lhy1024) [inline tools/pd-ctl/main.go]: Is it appropriate to output subcommands option in help ?

**Comment by disksing** [comment]
I think you can add some tests in `tests/pdctl/store`.

**Comment by shafreeck** [comment]
> I think you can add some tests in `tests/pdctl/store`.

Good idea ~

File: tools/pd-ctl/main.go (hunk 2 above)
Comment (shafreeck) [reply tools/pd-ctl/main.go]: It should be. The original implementation I think is not so correct. We use `pflag`  and `cobra` together to parse args, but `cobra` self is enough in fact. The `--help` message of pflag we set is too simple and override the help message from cobra which is more friendly.

```
> pd-ctl --help // print help message for pd-ctl
> pd-ctl store --help // print help message for store
```

File: tools/pd-ctl/pdctl/command/store_command.go (hunk 1 above)
Comment (shafreeck) [reply tools/pd-ctl/pdctl/command/store_command.go]: Why do I have so many typos 😂

File: tools/pd-ctl/pdctl/command/store_command.go [hunk 3]
```diff
@@ -31,14 +31,15 @@ var (
 // NewStoreCommand return a stores subcommand of rootCmd
 func NewStoreCommand() *cobra.Command {
 	s := &cobra.Command{
-		Use:   `store [delete|label|weight|limit] <store_id> [--jq="<query string>"]`,
-		Short: "show the store status",
+		Use:   `store [command] [flags]`,
+		Short: "manipulate stores",
```
Comment (disksing) [inline tools/pd-ctl/pdctl/command/store_command.go]: There are readonly commands, `store` or `store 1` do not manipulate stores.

Comment (shafreeck) [reply tools/pd-ctl/pdctl/command/store_command.go]: How about `manipulate or query stores`?

Comment (disksing) [reply tools/pd-ctl/pdctl/command/store_command.go]: Good idea.

File: tools/pd-ctl/main.go (hunk 2 above)
Comment (lhy1024) [reply tools/pd-ctl/main.go]:
```
./pd-ctl store --help                                                                       

manipulate or query stores

Usage:
  pd-ctl store [command] [flags]
  pd-ctl store [command]

Available Commands:
  delete           delete the store
  label            set a store's label value
  limit            set a store's rate limit
  remove-tombstone remove tombstone record if only safe
  weight           set a store's leader and region balance weight

Flags:
      --jq string   jq query

Global Flags:
      --cacert string   The path of file that contains list of trusted SSL CAs.
      --cert string     The path of file that contains X509 certificate in PEM format.
  -d, --detach          Run pdctl without readline. (default true)
  -h, --help            Help message.
  -i, --interact        Run pdctl with readline.
      --key string      The path of file that contains X509 key in PEM format.
  -u, --pd string       pd address (default "http://127.0.0.1:2379")
  -V, --version         Print version information and exit.

Use "pd-ctl store [command] --help" for more information about a command.
```

Comment (lhy1024) [reply tools/pd-ctl/main.go]: It seem output global flags too.

Comment (lhy1024) [reply tools/pd-ctl/main.go]: And it echo other redundant flags `--jq string   jq query`.

File: tools/pd-ctl/pdctl/command/store_command.go [hunk 4]
```diff
@@ -279,16 +295,28 @@ func setStoreWeightCommandFunc(cmd *cobra.Command, args []string) {
...
-		cmd.Println("Usage: store limit <store_id> <rate>")
+		cmd.Usage()
 		return
 	}
 	rate, err := strconv.ParseFloat(args[1], 64)
 	if err != nil || rate < 0 {
 		cmd.Println("rate should be a number that >= 0.")
 		return
 	}
+	// if the storeid is "all", set limits for all stores
```
Comment (lhy1024) [inline tools/pd-ctl/pdctl/command/store_command.go]:
```suggestion
	// if the store id is "all", set limits for all stores
```

File: tools/pd-ctl/main.go (hunk 2 above)
Comment (shafreeck) [reply tools/pd-ctl/main.go]: > It seem output global flags too.

Yes, it should print the local and global flags.

> And it echo other redundant flags `--jq string jq query`.

There is only one `--jq` here, why do you think it is redundant?

**Pull request merged**